src/scope/component-ops/traverse-versions.ts

```diff
@@ -14,7 +14,18 @@ import { HeadNotFound, ParentNotFound, VersionNotFound } from '../exceptions';
 import { ModelComponent, Version } from '../models';
```

Comment:
these changes are from another PR that's already merged to master... Due to overzealous re-basing on my part